curvefs/src/mds/topology/topology.h

```diff
@@ -200,23 +201,24 @@ class Topology {
                                                       [](const Partition &) {
                                                           return true;
                                                       }) const = 0;
+
     virtual std::list<Partition> GetPartitionInfosInPool(
         PoolIdType poolId, PartitionFilter filter = [](const Partition &) {
             return true;
         }) const = 0;
 
-    // choose randomly
-    virtual TopoStatusCode ChooseSinglePoolRandom(
-        PoolIdType *out,
-        const std::set<PoolIdType> &unavailablePools) const = 0;
-
     virtual TopoStatusCode ChooseZonesInPool(
         PoolIdType poolId, std::set<ZoneIdType> *zones,
         const std::set<ZoneIdType> &unavailableZones, int count) const = 0;
 
     virtual TopoStatusCode ChooseSingleMetaServerInZone(
         ZoneIdType zoneId, MetaServerIdType *metaServerId,
```

Comment:
replace these in recover schedule, and delete these code.

Reply:
> replace these in recover schedule, and delete these code.

fixed